lib/tasks/update_resource_quantity.rake

```diff
@@ -0,0 +1,14 @@
+namespace :resources do
+  desc "Update resources with nil quantity"
+  task update_resource_quantity: :environment do
+    Resource.all.each do |r|
+      if r.quantity.nil? || r.used.nil?
+        if r.used.nil?
+          Resource.update(r.id, quantity: 0, used: 0)
```

Comment:
And to make sure this works, you should create resources in master, then go into your branch and execute this task to make sure things become the way they are supposed to, and nothing else is missing.